ludwig/modules/recurrent_modules.py

```diff
@@ -35,33 +35,16 @@
 class RecurrentStack(LudwigModule):
     def __init__(
             self,
-            input_size=None,
-            hidden_size=256,
-            cell_type='rnn',
-            sequence_size=None,
-            num_layers=1,
-            bidirectional=False,
-            activation='tanh',
-            nonlinearity='tanh',
-            recurrent_activation='signmoid',
-            use_bias=True,
-            unit_forget_bias=True,
-            weights_initializer='xavier_uniform',
-            recurrent_initializer='orthogonal',
-            bias_initializer='zeros',
-            weights_regularizer=None,
-            recurrent_regularizer=None,
-            bias_regularizer=None,
-            activity_regularizer=None,
-            # kernel_constraint=kernel_constraint,
-            # recurrent_constraint=recurrent_constraint,
-            # bias_constraint=bias_constraint,
-            dropout=0.0,
-            recurrent_dropout=0.0,
+            input_size: int = None,
```

Comment:
Could you add a unit test for `RecurrentStack` in `tests/ludwig/modules/test_recurrent_modules.py`?

Reply:
Done.